docs/entropy.md

```diff
@@ -0,0 +1,71 @@
+# Using the Firecracker entropy device
+
+## What is the entropy device
+
+An entropy device is a [`virtio-rng` device][1] that provides guests with
+"high-quality randomness for guest use". Guests issue requests in the form of a
+buffer that will be filled with random bytes from the device. The source of
+random bytes that the device will use to fill the buffers is an implementation
+decision.
+
+On the guest side, the kernel uses random bytes received through the device
+as an extra source of entropy. Moreover, the guest VirtIO driver exposes the
+`/dev/hwrng` character device. User-space applications can use this device to
+request random bytes from the device.
+
+## Firecracker implementation
+
+Firecracker offers the option of attaching a single `virtio-rng` device starting
+with version 1.4. Users can configure it through the `/entropy` API endpoint.
```

Comment:
do we need this hard link to the version?

Reply:
We don't. I just tend to like knowing when a feature has been added to a tool I'm using, without having to browse the changelog. Do you see a problem if we do mention it?

Reply:
Well, if it's mentioned in the doc, then the feature is highly likely to be already available to the person reading it, and they don't need to care when it was introduced. Moreover, it's probably the changelog that led them in the doc where the feature is described.
The problems are not very significant/likely: what if the feature doesn't make it to the main before 1.4 or what if we need to cherry pick it to older releases? What if 1.4 is buggy, and 1.4.42 is the first working 1.4.* version?
It's certainly a nit though.

Reply:
Ok, I agree that the problems, even though not very likely, will be annoying if we have to deal with and it's easy to avoid. I removed the version mention.